### PR TITLE
feat: add user summary v2

### PR DESCRIPTION
**Pull request by DawoudSheraz**
### Description
Add User Summary/Account information v2 component. Besides the component addition, a few updates are:

- Fixing propType warning for account actions
- Updating one FBE component test
- Creating a new wrapper component for account actions

### Screenshots
Screenshot 2021-09-28 at 4 28 20 PM
Screenshot 2021-09-28 at 4 28 33 PM

Screenshot 2021-09-28 at 4 28 05 PM

### Testing Instructions/Pointers
- Uncomment usersv2 route from index page
- Add a valid username/email info
- manually edit any value to be a much larger value and see if there are any overflow issues
- To check activation key, create a user manually from LMS via Sign up. Once created, change their active status to False from admin.

### CODE REVIEW

File: src/overrides.scss [hunk 1]
```diff
@@ -5,8 +5,8 @@
   color: black;
   .nav-link.active {
     border-color: #fff;
-    color: #00262B;
-    border-bottom-color: #00262B;
+    color: #00262b;
```
Comment (azanbinzahid) [inline src/overrides.scss]: [sugg] can we define variables for colors for consistency?

File: src/users/v2/UserSummary.jsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  changeHandler,
+}) {
+  return (
+    <section className="mb-3">
+      <div className="d-flex flex-row flex-wrap">
+        <div className="col-sm-6">
+
+          <div id="account-table" className="flex-column pr-4 m-3 card account-info">
+            <h3>Account Details</h3>
+            <table>
```
Comment (azanbinzahid) [inline src/users/v2/UserSummary.jsx:20]: can't we use TableV2 somehow?

File: src/users/v2/UserSummary.jsx [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      passwordToggleHistory: PropTypes.arrayOf(PropTypes.shape(
+        {
+          created: PropTypes.string.isRequired,
+          comment: PropTypes.string.isRequired,
+          disabled: PropTypes.bool.isRequired,
+          createdBy: PropTypes.string.isRequired,
+        },
+      )),
+    }),
+  }),
```
Comment (azanbinzahid) [inline src/users/v2/UserSummary.jsx]: can add `isRequired` here and remove it from default prop.

File: src/users/v2/UserSummary.test.jsx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+  afterEach(() => {
+    wrapper.unmount();
+  });
+
+  describe('Default Correct Data Values', () => {
+    it('User Data Values', () => {
+      const ComponentUserData = wrapper.prop('userData');
+      const ExpectedUserData = UserSummaryData.userData;
+
+      expect(ComponentUserData.id).toEqual(ExpectedUserData.id);
```
Comment (azanbinzahid) [inline src/users/v2/UserSummary.test.jsx]: we may use loop over all object keys

File: src/users/v2/UserSummary.jsx (hunk 2 above)
Comment (DawoudSheraz) [reply src/users/v2/UserSummary.jsx:20]: The problem here was that TableV2 displays the headers horizontally. Even with the previous Table, a hack/patch was used, where there are only 2 columns, Label and Value. This didn't work with TableV2.

File: src/overrides.scss (hunk 1 above)
Comment (DawoudSheraz) [reply src/overrides.scss]: sure.

**Comment by azanbinzahid** [comment]
Tested locally, can we move show history modal to collapsible table inside actions? Or, fix width of the modal.

**Comment by DawoudSheraz** [comment]
> Tested locally, can we move show history modal to collapsible table inside actions? Or, fix width of the modal.

ok, I will take a look at the width.

**Review by azanbinzahid — APPROVED** [review]

**Review by ansabgillani — APPROVED** [review]

**Pull request merged**